internal/ingest/ingest_test.go

```diff
@@ -488,22 +488,55 @@ func TestReSyncWithDepth(t *testing.T) {
 	wait, err := te.ingester.Sync(context.Background(), te.pubHost.ID(), te.pubHost.Addrs()[0], 1, false)
 	require.NoError(t, err)
 	<-wait
-	allAds := typehelpers.AllMultihashesFromAdLink(t, adHead, te.publisherLinkSys)
-	require.NoError(t, checkAllIndexed(te.ingester.indexer, te.pubHost.ID(), allAds[2:]))
-	require.Error(t, checkAllIndexed(te.ingester.indexer, te.pubHost.ID(), allAds[0:1]))
+	allMHs := typehelpers.AllMultihashesFromAdLink(t, adHead, te.publisherLinkSys)
+	require.NoError(t, checkAllIndexed(te.ingester.indexer, te.pubHost.ID(), allMHs[2:]))
+	require.Error(t, checkAllIndexed(te.ingester.indexer, te.pubHost.ID(), allMHs[0:1]))
```

Comment:
```suggestion
	require.Error(t, checkAllIndexed(te.ingester.indexer, te.pubHost.ID(), allMHs[0:2]))
```

The test is actually incorrect because `allMHs[2:]` is a zero-length slice. Add
```go
{ChunkCount: 1, EntriesPerChunk: 1, EntriesSeed: 3},
```
to `EntryChunkBuilders`

Reply:
I'll just fix it to just need the two ads.